src/redux/actions/concepts/dictionaryConcepts.js

```diff
@@ -369,3 +369,15 @@ export const updateConcept = (conceptUrl, data, history) => async (dispatch) =>
 export const clearPreviousConcept = () => (dispatch) => {
   dispatch({ type: CLEAR_PREVIOUS_CONCEPT });
 };
+
+export const unretireMapping = url => async (dispatch) => {
+  dispatch(isFetching(true));
+  try {
+    const response = await instance.put(url, { retired: false });
```

Comment:
How is this related to the ticket?

Reply:
When a concept mapping is removed, the concept is actually retired. For that specific concept mapping to be re-added, it has to be unretired i.e the retired flag has to be updated to false.